src/test/java/com/monumental/models/unittest/MonumentUnitTests.java

```diff
@@ -192,31 +180,31 @@ public void testMonument_validate_New_NotNullTitle_NullMaterial() {
         List<String> violationMessages = result.getViolationMessages();
 
         assertEquals(1, violationMessages.size());
-        assertTrue(violationMessages.contains("Material can not be null"));
+        assertTrue(violationMessages.contains("ID can not be specified on insert"));
     }
 
-    @Test
-    public void testMonument_validate_New_NotNullId_NotNullTitle_NullMaterial() {
-        Monument monument = new Monument();
-        monument.setId(1);
-        monument.setTitle("Title");
+     @Test
+     public void testMonument_validate_Existing_NullId_NotNullTitle() {
+     Monument monument = new Monument();
+     monument.setTitle("Title");
 
-        Monument.ValidationResult result = monument.validate(Monument.New.class);
+     Monument.ValidationResult result = monument.validate(Monument.Existing.class);
 
-        assertFalse(result.isValid());
+     assertFalse(result.isValid());
 
-        List<String> violationMessages = result.getViolationMessages();
+     List<String> violationMessages = result.getViolationMessages();
 
-        assertEquals(2, violationMessages.size());
-        assertTrue(violationMessages.contains("Material can not be null"));
-        assertTrue(violationMessages.contains("ID can not be specified on insert"));
-    }
+     assertEquals(1, violationMessages.size());
+     assertTrue(violationMessages.contains("ID can not be null on update"));
+     }
+     */
 
+    /**
+     * TODO: Replace this with repositories instead of services
```

Comment:
What services are we using here...?

Reply:
I think I copied this from another place lol and now I can't remember what's wrong with this one